strata_period_wrangler.py

```diff
@@ -45,8 +45,13 @@ def lambda_handler(event, context):
     logger = logging.getLogger("Strata")
     logger.setLevel(10)
 
+    # Define run_id outside of try block
+    run_id = 0
     try:
         logger.info("Strata Wrangler Begun")
+        # Retrieve run_id before input validation
+        # Because it is used in exception handling
+        run_id = event['RuntimeVariables']['id']
```

Comment:
Didn't we decide upon 'run_id' over 'id' in the end?

Reply:
quite right, i'll get them altered before starting enrichment.